src/sqlancer/doris/gen/DorisRandomQuerySynthesizer.java

```diff
@@ -0,0 +1,70 @@
+package sqlancer.doris.gen;
+
+import sqlancer.Randomly;
+import sqlancer.common.ast.newast.Node;
+import sqlancer.common.ast.newast.TableReferenceNode;
+import sqlancer.doris.ast.DorisConstant;
+import sqlancer.doris.ast.DorisExpression;
+import sqlancer.doris.ast.DorisJoin;
+import sqlancer.doris.ast.DorisSelect;
+import sqlancer.doris.DorisProvider.DorisGlobalState;
+import sqlancer.doris.DorisSchema.DorisTable;
+import sqlancer.doris.DorisSchema.DorisTables;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class DorisRandomQuerySynthesizer {
+
+    private DorisRandomQuerySynthesizer() {
+    }
+
+    public static DorisSelect generateSelect(DorisGlobalState globalState, int nrColumns) {
```

Comment:
I believe this class is not yet used? If so, let's remove it from this initial PR.